packages/manfred2html/src/engine/cv-monochrome-force/html-parts/about-me-section/about-me-section.ejs

```diff
@@ -0,0 +1,6 @@
+<%_ if (profile && profile !== 'undefined' ) { -%>
+    <section class="section about">
```

Comment:
description property is optional, if we don't have it we sould not display this section